packages/expo-clipboard/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- Deprecated `setString`. Use `setStringAsync` instead. ([#16320](https://github.com/expo/expo/pull/16320) by [@barthap](https://github.com/barthap))
```

Comment:
Deprecating is imho not a breaking change — as a result of this change users don't have to modify their code yet. You could use another changelog category, like `⚠️ Notices`

Reply:
You're right, I had similar thoughts, but I checked that deprecations in other changelogs were put in the `Breaking changes` section.

But I can move that if you wish 😁

Reply:
If they are put there, it's wrong 😅 Refer to [Updating Changelogs](https://github.com/expo/expo/blob/main/guides/contributing/Updating%20Changelogs.md#categories) guide.

Reply:
Great 😁  I changed this in https://github.com/expo/expo/commit/1ca5a5daab190970e11873e18adea86c2e90e2e9